addons/expanded-backpack/backpack.css

```diff
@@ -0,0 +1,31 @@
+.backpack_backpack-list_MOp5X {
+    /* set height of the backpack picker, 5.5rem is the height of 1 row so multiply it by the rows setting */
+    min-height: calc(var(--expandedBackpack-rows) * 5.5rem);
+}
+
+.backpack_backpack-list-inner_10a2A {
+    /* move backpack items to the top left instead of vertically centering them */
+    -webkit-align-items: start;
+        -ms-align-items: start;
```

Comment:
There's an unprefixed CSS property name supported in all modern browsers: `align-items`